src/network/network_server.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
:(

Reply:
Ah, so that's where that wayward carriage return ended up :thinking: